.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ concurrency:
 
 env:
   CARGO_INCREMENTAL: 0
-  RUSTFLAGS: '-Dwarnings'
+  RUSTFLAGS: "-Dwarnings"
```

Comment:
This is my formatter - should I revert? Add it to CI?

Reply:
Add it to CI :) Not necessarily on this PR but we already did that on Wind Tunnel